src/components/dashboard/index.js

```diff
@@ -163,7 +165,7 @@ const cleanDescription = (description) => {
 export const DashboardItem = (props) => {
     const classes = useStyles();
     const { kind, content } = props;
-    const cardID = `${kind}-${content.id}`;
+    const cardID = `${kind}.${content.id}`;
```

Comment:
This could just be `buildID(kind, content.id)`

Reply:
Also fixed. Thanks for catching this.